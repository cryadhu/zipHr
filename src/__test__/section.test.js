import sections from "../utils/sections.json";

test("should contain list of strings(sections)", () => {
  expect(sections).toMatchSnapshot();
});
