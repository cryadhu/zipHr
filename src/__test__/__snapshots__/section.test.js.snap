// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should contain list of strings(sections) 1`] = `
Array [
  "arts",
  "automobiles",
  "books",
  "business",
  "fashion",
  "food",
  "health",
  "home",
  "insider",
  "magazine",
  "movies",
  "nyregion",
  "obituaries",
  "opinion",
  "politics",
  "realestate",
  "science",
  "sports",
  "sundayreview",
  "technology",
  "theater",
  "t-magazine",
  "travel",
  "upshot",
  "us",
  "world",
]
`;
